src/include/job.h

```diff
@@ -1161,33 +1161,21 @@ extern int   do_tolerate_node_failures(job *);
  */
 #ifdef PBS_MOM
 
-extern job  *job_recov_fs(char *);
-extern int   job_save_fs(job *, int);
-extern int   job_or_resv_save_fs(void *, int, int);
-void*	job_or_resv_recov_fs(char *, int);
+extern job *job_recov_fs(char *);
+extern int job_save_fs(job *, int);
 #define job_recov job_recov_fs
-#define job_save job_save_fs
-#define job_or_resv_save job_or_resv_save_fs
-#define job_or_resv_recov job_or_resv_recov_fs
+#define job_save  job_save_fs
 
 #else
 
-extern job  *job_recov_db(char *);
-extern void *job_or_resv_recov_db(char *, int);
-extern int  job_save_db(job *, int);
-extern int   job_or_resv_save_db(void *, int, int);
+extern job *job_recov_db(char *);
```

Comment:
Yes, true, these changes are directly related to avltree refactoring but I needed these changes so that I can put pbs_idx_insert() in resv_alloc() (as per [this](https://github.com/openpbs/openpbs/pull/1730#discussion_r433720657))

I am happy to remove them, but then either I have to wait for you PR to go in first, or we have to do resv_tree_op() stuff like in my old PR (like [this](https://github.com/openpbs/openpbs/pull/1730/files#diff-f41edbf0d83389af469d191a7aa5b195R848))

Reply:
yeah u can remove them, my PR should get in soon